src/lib/extract-acl-context.ts

```diff
@@ -32,15 +28,37 @@ const getPermissions = (config: RuntimeConfig, request: FastifyRequest): string[
   return parsedProperties.permissions ?? []
 }
 
-export const extractAclContext = (
+const cleanHeaders = (headers: Record<string, string | string[] | undefined>): Record<string, string | string[] | undefined> => {
+  delete headers.cookie
+  return headers
+}
+
+export const extractAclContext = async (
   config: RuntimeConfig,
   request: FastifyRequest
-): AclContext => {
+): Promise<string[]> => {
+  if (config.ACL_CONTEXT_BUILDER !== undefined) {
+    const aclContext = await config.ACL_CONTEXT_BUILDER({
+      headers: cleanHeaders(request.headers),
```

Comment:
Removing `sensitive` headers was an idea before employing a sandbox. i reckon that now it ain't that much necessary anymore. beside it is quite arbitrary to wipe only the `cookie`. why not `authorization` then?

The user cannot access `fetch` now within the sandbox hence it should be harmless to pass the whole request context to the sandbox

Reply:
If we recognize logging as harmless then I agree with you passing the whole request: a malicious user can make the function return a `cookie`/`authorization` header, then the middleware may log it in case of errors (https://github.com/micro-lc/middleware/blob/comp-307/extract-generic-acl-context/src/sdk/evaluate-acl.ts#L70)

Reply:
inside the `sandbox` there's no `console` defined

Reply:
I know, but the outcome of the sandboxed function is logged in [some cases](https://github.com/micro-lc/middleware/blob/comp-307/extract-generic-acl-context/src/sdk/evaluate-acl.ts#L70)